react/src/components/BAITable.tsx

```diff
@@ -91,26 +91,28 @@ const ResizableTitle = (
   );
 };
 
-interface BAITableProps extends TableProps {
+interface BAITableProps<RecordType extends object = any>
+  extends Omit<TableProps<RecordType>, 'components'> {
   resizable?: boolean;
+  components?: TableProps<RecordType>['components'];
```

Comment:
Is the reason for declaring the components in Antd's TableProps<recordtype> as they are, to explicitly show the inputs that the component requires?

Reply:
I forgot to remove it. Thank you. I have just removed it and submitted.